byron/ledger/executable-spec/src/Ledger/Update.hs

```diff
@@ -1616,3 +1656,107 @@ protocolVersionEndorsementGen upienv upistate =
         endorsementsMap = Set.toList (endorsements upistate)
                         & fmap (second Set.singleton)
                         & Map.fromListWith Set.union
+
+--------------------------------------------------------------------------------
+-- Goblins instances
+--------------------------------------------------------------------------------
+
+instance GeneOps g => Goblin g Metadata where
```

Comment:
I guess this one can't be derived. Do you know why?

Reply:
I added another case to the TH derivation functions to handle no-argument constructors, so this can now be derived.